src/modules/FFmpeg/FFDecSW.cpp

```diff
@@ -339,8 +339,6 @@ bool FFDecSW::open(StreamInfo &streamInfo, VideoWriter *)
 		return false;
 	if (codec_ctx->codec_type == AVMEDIA_TYPE_VIDEO)
 	{
-		if (codec->capabilities & CODEC_CAP_DR1)
```

Comment:
> ... I.e. this flag is now always set.

So this looks like FFmpeg 2.2.x bug :) The best way is to compile & test :smiley: